packages/@lwc/engine-core/src/framework/restrictions.ts

```diff
@@ -91,7 +94,7 @@ export function patchElementWithRestrictions(elm: Element, options: { isPortal:
     };
 
     // Apply extra restriction related to DOM manipulation if the element is not a portal.
-    if (isFalse(options.isPortal)) {
+    if (isFalse(options.isLight) && isFalse(options.isPortal)) {
```

Comment:
Nit: We agreed in the past that `isTrue/isFalse` should only be used in place of expressions like `foo === true` or `bar === false`. Since `isLight` and `isPortal` are boolean-typed and we don't usually use `===` when we know the thing is a boolean, we wouldn't have used this utility function here.

Than being said, I don't think we've been following this guideline consistently so I'm fine either way :)

Reply:
I'm happy to replace this with just `!options.isLight && !options.isPortal`. And thanks for explaining what `isTrue`/`isFalse` did; I didn't quite get it before. :)